io/www/package.json

```diff
@@ -13,20 +13,27 @@
     "uuid": "^8.0.0"
   },
   "devDependencies": {
+    "@istanbuljs/nyc-config-typescript": "^1.0.2",
     "chai": "^4.3.6",
     "eslint": "^8",
     "esm": "^3.2.25",
     "mocha": "^10.2.0",
     "nock": "^13.0.0",
-    "nyc": "^15.1.0"
+    "nyc": "^15.1.0",
+    "rollup": "^3.29.4",
+    "@rollup/plugin-commonjs": "^25.0.4",
+    "@rollup/plugin-node-resolve": "^15.2.1",
+    "@rollup/plugin-terser": "^0.4.4",
+    "@rollup/plugin-json": "^6.1.0"
   },
   "scripts": {
-    "test": "nyc --check-coverage --lines=99 --functions=99 --branches=99 --statements=99 mocha --recursive ./test",
+    "test": "nyc --exclude=src/fragment/odinSchemaTransform.js --exclude=src/ost-products/read.js --exclude=src/ost-products/write.js mocha --recursive ./test",
     "test:watch": "mocha --watch --recursive ./test",
     "test:file": "mocha --watch --recursive --grep",
-    "test:coverage": "nyc --reporter=lcov --reporter=text-summary --check-coverage --lines=99 --functions=99 --branches=99 --statements=99 npm run test",
+    "test:coverage": "nyc --exclude=src/fragment/odinSchemaTransform.js  --exclude=src/ost-products/read.js --exclude=src/ost-products/write.js npm run test",
     "lint": "eslint --ignore-pattern web-src --no-error-on-unmatched-pattern test src actions",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "build:client": "rollup -c rollup.config.cjs"
```

Comment:
so our new approach is when i do a change in /mas/io/www I also need to run 'build:client' and commit result of that build? to ensure feature parity of Preview experience?

Reply:
yes! i was planning to add an hashsum check in io check but later

Reply:
check https://github.com/adobecom/mas/pull/259